flink-jpmml-scala/src/main/scala/io/radicalbit/flink/pmml/scala/api/exceptions/package.scala

```diff
@@ -21,6 +21,32 @@ package io.radicalbit.flink.pmml.scala.api
 
 package object exceptions {
 
+  /** Models conformity failure between PMML model and input [[org.apache.flink.streaming.api.scala.DataStream]]
+    *
+    * @param msg
+    */
+  private[scala] class InputValidationException(msg: String) extends RuntimeException(msg)
+
+  /** Models [[org.jpmml.evaluator.EvaluatorUtil.prepare()]] method failure
+    *
+    * @param msg
+    */
+  private[scala] class InputPreparationException(msg: String) extends RuntimeException(msg)
+
+  /** Models empty result from [[org.jpmml.evaluator.ModelEvaluator]] evaluation
+    *
+    * @param msg
+    */
+  private[scala] class JPMMLExtractionException(msg: String) extends RuntimeException(msg)
+
+  /** Models failure on loading PMML model from distributed system
+    *
+    * @param msg
+    * @param throwable
+    */
+  private[scala] class ModelLoadingException(msg: String, throwable: Throwable)
+      extends RuntimeException(msg, throwable)
+
   /** Prediction failure due to [[io.radicalbit.flink.pmml.scala.api.EmptyEvaluator]]
     *
     * @param msg
```

Comment:
Please, can you remove `@param` bullets if not specified? Herein it doesn't make so much sense to insert them.